apps/api/src/utils/routes.ts

```diff
@@ -130,3 +135,31 @@ export const aliasApplicationContentsRecord = (update: UpdateEditApplicationRequ
 
 	return formatedUpdate;
 };
+
+/**
+ * Helper function to convert Postgres snake_case to FE camelCase for CollaboratorRecord
+ * @param data type CollaboratorRecord in snake_case
+ * @returns  type GetCollaboratorsResponse in camelcase
+ */
+
+export const aliasCollaboratorRecord = (data: CollaboratorRecord[]): GetCollaboratorsResponse[] => {
+	const formatedUpdate: GetCollaboratorsResponse[] = [];
+
+	data.forEach((value) => {
+		formatedUpdate.push({
+			id: value.id,
+			applicationId: value.application_id,
+			collaboratorFirstName: value.first_name,
+			collaboratorMiddleName: value.middle_name,
+			collaboratorLastName: value.last_name,
+			collaboratorInstitutionalEmail: value.institutional_email,
+			collaboratorPositionTitle: value.position_title,
+			collaboratorPrimaryAffiliation: value.title,
+			collaboratorResearcherProfileURL: value.profile_url,
+			collaboratorSuffix: value.suffix,
+			collaboratorType: value.collaborator_type,
+		});
+	});
+
+	return formatedUpdate;
```

Comment:
Probably want to correct the spelling here also! Should be `formattedUpdate`:

```suggestion
	const formattedUpdate: GetCollaboratorsResponse[] = [];

	data.forEach((value) => {
		formattedUpdate.push({
			id: value.id,
			applicationId: value.application_id,
			collaboratorFirstName: value.first_name,
			collaboratorMiddleName: value.middle_name,
			collaboratorLastName: value.last_name,
			collaboratorInstitutionalEmail: value.institutional_email,
			collaboratorPositionTitle: value.position_title,
			collaboratorPrimaryAffiliation: value.title,
			collaboratorResearcherProfileURL: value.profile_url,
			collaboratorSuffix: value.suffix,
			collaboratorType: value.collaborator_type,
		});
	});

	return formattedUpdate;
```